examples/tutorials/AlphafoldXDeepChem.ipynb

```diff
@@ -0,0 +1,1626 @@
+{
```

Comment:
<p>Combine these lines into a paragraph. Make sure to use complete sentences, active voice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>